modules/base/humanitec-operator.tf

```diff
@@ -0,0 +1,138 @@
+# Installs the humanitec-operator into the cluster
+
+# More details https://developer.humanitec.com/integration-and-extensions/humanitec-operator/overview/
+
+resource "kubernetes_namespace" "humanitec_operator" {
+  metadata {
+    labels = {
+      "app.kubernetes.io/name"     = "humanitec-operator"
+      "app.kubernetes.io/instance" = "humanitec-operator"
```

Comment:
Could we add `"pod-security.kubernetes.io/enforce" = "restricted"`, I tested it, we support it, let's show it?